config/locales/en.yml

```diff
@@ -196,6 +196,8 @@ en:
       long: ! '%B %d, %Y %H:%M'
       short: ! '%d %b %H:%M'
     pm: pm
+  placeholder:
+    full_name_or_nickname_or_event_name: "Full Name or Nickname or Event Name"
```

Comment:
Right, that makes sense. I had put it in a subsection to be consistent with the heirarchical structure of ``locales/en.yml`` and also because there would be another placeholder for the reimbursement search box. But since we have put it in ``application.en.yml``, it seems better not to use a new subsection, for now.